platform/android/MapboxGLAndroidSDK/build.gradle

```diff
@@ -24,29 +20,23 @@ dependencies {
 }
 
 android {
-    compileSdkVersion Integer.parseInt(project.ANDROID_BUILD_SDK_VERSION)
-    buildToolsVersion project.ANDROID_BUILD_TOOLS_VERSION
+    compileSdkVersion rootProject.ext.compileSdkVersion
```

Comment:
@tobrun Why not doing this too with all compile dependencies like MAS or `compile 'com.squareup.okhttp3:okhttp:3.5.0'`, `compile 'com.mapzen.android:lost:2.0.0'`, `compile 'com.jakewharton.timber:timber:4.3.1'`?